gammapy/background/tests/test_models.py

```diff
@@ -43,3 +50,97 @@ def test_y_model(self):
         model = self.model.y_model(-30)
         assert isinstance(model, Gaussian1D)
         assert_allclose(model.parameters, [0, -1, 0.4])
+
+
+class TestCubeBackgroundModel():
+
+    @remote_data
+    def test_read_bin_table(self):
+
+        # test shape of bg cube when reading a file
+        filename = datasets.get_path('../test_datasets/background/bg_cube_model_test.fits',
+                                     location='remote')
+        bg_cube_model = CubeBackgroundModel.read(filename, format='table')
+        assert len(bg_cube_model.background.shape) == 3
+        assert bg_cube_model.background.shape == (len(bg_cube_model.energy_bins) - 1,
+                                                  len(bg_cube_model.dety_bins) - 1,
+                                                  len(bg_cube_model.detx_bins) - 1)
+
+    def test_image_plot(self):
```

Comment:
To get rid of this fail:
https://travis-ci.org/gammapy/gammapy/jobs/71722967#L991

you have to add a decorator as in this example:
https://github.com/gammapy/gammapy/blob/9c979c56ed278b7deda1cada1788241be573e577/gammapy/time/tests/test_plot.py#L15